examples/sse/index.js

```diff
@@ -0,0 +1,110 @@
+"use strict";
+
+const path = require("node:path");
+const { ServiceBroker, Errors } = require("moleculer");
+const ApiGatewayService = require("../../index");
+const ChatService = require("./chat.service");
+
+const { MoleculerError } = Errors;
+
+const SSE_RETRY_TIMEOUT = 15000; // 15 seconds
+const PORT = 3000;
+const HOST = "0.0.0.0";
+const SSE_HEADERS = {
+	Connection: "keep-alive",
+	"Content-Type": "text/event-stream",
+	"Cache-Control": "no-cache",
+};
+
+// Create broker
+const broker = new ServiceBroker({
+	logger: console,
+	metrics: true,
+	validation: true,
+});
+
+broker.createService(ChatService);
+
+// Load API Gateway
+broker.createService({
+	name: "sse.gateway",
+	mixins: [ApiGatewayService],
+	settings: {
+		port: PORT,
+
+		ip: HOST,
+
+		assets: {
+			folder: path.join(__dirname, "assets"),
+		},
+
+		routes: [
+			{
+				path: "/api/chat",
+				aliases: {
+					"POST message": "chat.postMessage",
+					"GET message"(request, response) {
+						response.writeHead(200, SSE_HEADERS);
+						response.$service.addSSEListener(
+							response,
+							"chat.message"
+						);
+					},
+				},
+			},
+		],
+	},
+
+	events: {
+		"**"(context) {
```

Comment:
This event subscription is not recommended, it will catch a lot of internal events as well. Please use more exact event name.